server/tools/cli/databases/mssql/change-database.cli

```diff
@@ -1,5 +1,5 @@
 /subsystem=datasources/data-source=KeycloakDS: remove()
-/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url="jdbc:sqlserver://${env.DB_ADDR:mssql}:${env.DB_PORT:1433};databaseName=${env.DB_DATABASE:keycloak};sendStringParametersAsUnicode=false;user=${env.DB_USER:keycloak};password=${env.DB_PASSWORD:password};${env.JDBC_PARAMS:}", driver-name=sqlserver)
+/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url="jdbc:sqlserver://${env.DB_ADDR:mssql}:${env.DB_PORT:1433};databaseName=${env.DB_DATABASE:keycloak};sendStringParametersAsUnicode=false;${env.JDBC_PARAMS:}", driver-name=sqlserver)
```

Comment:
This seems to me like a non-backwards compatible change. If someone used `env.DB_USER` and `env.DB_PASSWORD`, this PR would break his code, wouldn't it?

Reply:
Please have a look at line 3 and 4, where username and password are set directly.

`/subsystem=datasources/data-source=KeycloakDS: write-attribute(name=user-name, value=${env.DB_USER:keycloak})
/subsystem=datasources/data-source=KeycloakDS: write-attribute(name=password, value=${env.DB_PASSWORD:password})`

It does not really matter even if you constructed your URL with dummy username and password like so:

`/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url="jdbc:sqlserver://${env.DB_ADDR:mssql}:${env.DB_PORT:1433};databaseName=${env.DB_DATABASE:keycloak};sendStringParametersAsUnicode=false;user=dummy;password=dummy;${env.JDBC_PARAMS:}", driver-name=sqlserver)`

It would still work and be backward compatible, because username and password set on line 3 and 4 override whatever username and password was provided by URL. As far as know having username and password in URL and set it directly as well does not serve any purpose, it is just confusing.

Reply:
This change is consistent with how `change-database.cli` scripts for all other databases work.
I have checked all of them and none of them provides username and password in connection url.

In case you are in doubt you can build it locally and try to run `keycloak-mssql.yml` from `docker-compose-examples`... it should work as before.

The only difference being, that now you should be able to use passwords which contain characters like "}{".
(There are 3 places where you need to change password in `keycloak-mssql.yml`.)

Reply:
Ok, thanks for the explanation.